constellations/gemini-user/src/achievements/achievements.sass

```diff
@@ -1,2 +1,61 @@
 #view-achieve h1
-  color: red
+	color: red
+
+
+.achieve-card
+	margin-bottom: 96px
+	
+	& h2
+		line-height: 30px
+		font-size: 18px
+		color: #056571
+		border-bottom: 1px solid #222
+
+	& li
+		height: 96px
+		line-height: 24px
+		border-bottom: 1px solid #b8b8b8
+		font-size: 16px
+		
+		display: flex
+		flex-direction: row
+		align-items: center
```

Comment:
Same thing here with the ordering; positioning should be first, then display, then color and text.